src/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore/DatabaseErrorPageOptions.cs

```diff
@@ -12,6 +14,26 @@ namespace Microsoft.AspNetCore.Builder
     public class DatabaseErrorPageOptions
     {
         /// <summary>
+        /// Constructs a new DatabaseErrorPageOptions.
+        /// </summary>
+        public DatabaseErrorPageOptions() { }
+
+        /// <summary>
+        /// Constructs and configures a new DatabaseErrorPageOptions.
+        /// </summary>
+        /// <param name="configureOptions">The configuration actions that will be applied.</param>
+        public DatabaseErrorPageOptions(IEnumerable<IConfigureOptions<DatabaseErrorPageOptions>> configureOptions) : this()
+        {
+            if (configureOptions != null)
```

Comment:
would this throw argument null exception now, instead of being an `if {...}` safety?